.github/workflows/sdk-pr.yml

```diff
@@ -28,6 +28,11 @@ jobs:
           options: "--check"
           src: "."
 
+      - name: Run isort
+        # adds options `--check-only --diff` by default - https://pycqa.github.io/isort/docs/configuration/github_action.html
+        uses: jamescurtin/isort-action@master
```

Comment:
Rather than master can we have a tagged release? Reduces chances of breaking because the tool broker

Reply:
And check it still works after the change